internal/node_loader.js

```diff
@@ -68,14 +68,13 @@ function loadRunfilesManifest(manifestPath) {
   return new Promise((resolve, reject) => {
     try {
       const input = fs.createReadStream(manifestPath, {encoding: 'utf-8'});
+      input.on('error', reject);
       const manifest = readline.createInterface({input});
       manifest.on('line', line => {
         if (!line) return;
         const [runfilesPath, realPath] = line.split(" ");
         runfilesManifest[runfilesPath] = realPath;
-      }).on('close', () => {
-        resolve(runfilesManifest);
-      });
+      }).on('close', resolve);
```

Comment:
Maybe document that this is a void promise? loadRunfilesManifest sounds as if it should resolve with the manifest.

Reply:
Actually I don't like that it's a void promise. Changed to return the manifest, which starts out undefined, and is set before bootstrapping.